test/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/DeclarationConfirmationControllerSpec.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.merchandiseinbaggagefrontend.controllers
+
+import play.api.test.Helpers._
+import uk.gov.hmrc.merchandiseinbaggagefrontend.model.core.{DeclarationType, SessionId}
+import uk.gov.hmrc.merchandiseinbaggagefrontend.views.html.DeclarationConfirmationView
+
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class DeclarationConfirmationControllerSpec extends DeclarationJourneyControllerSpec {
+
+  val view = app.injector.instanceOf[DeclarationConfirmationView]
+  val controller = new DeclarationConfirmationController(controllerComponents, actionBuilder, view)
+
+  "on page load return 200" in {
```

Comment:
I can see it might have been useful during development but does it give us anything the selenium spec doesn't? 

If not maybe remove it now?